xsd/xsd.go

```diff
@@ -38,6 +39,107 @@ type Type interface {
 	isType()
 }
 
+func TypesEqual(a, b Type) bool {
```

Comment:
```suggestion
func typesEqual(a, b Type) bool {
```

Reply:
This needs to be exported, we call it from a different package